src/network/NetworkController.ts

```diff
@@ -258,13 +269,9 @@ export class NetworkController extends BaseController<
    * Refreshes the current network code.
    */
   async lookupNetwork() {
-    /* istanbul ignore if */
-    if (!this.ethQuery || !this.ethQuery.sendAsync) {
```

Comment:
If we mark `this.ethQuery` as an EthQuery object, then there's no way that it can't have `sendAsync`.